test/arrays/ferguson/if-expr/aif-param.chpl

```diff
@@ -0,0 +1,21 @@
+config param param_useA = true;
```

Comment:
I'm having trouble parsing the name of this file (and the similarly named ones).  Maybe add a comment at the top of what it means?